devimint/src/util.rs

```diff
@@ -52,13 +50,18 @@ impl Drop for ProcessHandleInner {
     }
 }
 
+#[derive(Clone)]
 pub struct ProcessManager {
-    pub globals: vars::Global,
+    pub globals: Arc<vars::Global>,
+    handles: Arc<Mutex<Vec<ProcessHandle>>>,
```

Comment:
incorrect: keeping a reference to ProcessHandle will prevent it from stopping on drop.

Reply:
So what? Shouldn't we always explicit kill them?

Reply:
Should we make it a `Vec<WeakProcessHandle>` (with `Weak` instead of `Arc`?)

Reply:
I liked the invariant that Bitcoind value existing means bitcoind is running, maybe it is not important?